tests/conftest.py

```diff
@@ -222,7 +222,7 @@ def unicode_emojis():
     'stream_id': 205,
     'subject': 'Test',
     'reactions': [],
-    'subject_links': [],
+    'topic_links': [],
```

Comment:
Note that we aim to support Zulip 2.1+, which is prior to feature level 1, so we need to support subject_links as well as topic_links.

Given the text below, you might consider whether we can refactor more tests to use the same message templates - so we don't need to make as many changes in multiple places.

Reply:
For now, I've added `topic_links` wherever we have `subject_links` so that there is no version support issue. This is keeping in mind my reply to the comment below.

Reply:
Makes sense. My suggestion would be that we let `subject_links` stay untouched for now in the tests and fixtures. We're validating for both `*_links` parameter in the helper of later commit, in such a way that only one could possibly exist (I don't know why I didn't consider that for this commit :laughing:)